File: Div2A/CF263-D2-A.cpp
#include <iostream>
#include <cmath>

int main() {
    int input, x,y, moves, i(1),j(1);

    while(i<=5) {
        j = 1;
        while(j<=5) {
            std::cin>>input;
            if(input == 1) {
                x = i;
                y = j;
            }
            j++;
        }
        i++;
    }

    moves = std::abs(x-3) + std::abs(y-3);
    std::cout<<moves;
}

File: Div2A/CF263-D2-A.py
m = 0
n = 0
for i in range(5):
    arr = [int(x) for x in input().split(" ")]
    try: 
        arr.index(1)
        find_1 = 1
    except:
        find_1 =0
    
    if find_1:
        m = i
        n = arr.index(1)

move = abs(2-m) + abs(2-n)
print(move)